app/controllers/flood_risk_engine/enrollments/steps_controller.rb

```diff
@@ -30,10 +30,18 @@ def update
         end
       end
 
+      def remove_exemption
+        @enrollment = Enrollment.find(params[:id])
```

Comment:
Even if we add token stuff  later is it worth replacing raw calls to `Enrollment.find(params[:id])` with a `set_enrollment` type method to make it easier to refactor later if needed ?